tools/istio-iptables/pkg/builder/iptables_builder_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package builder
+
+import (
+	"reflect"
+	"testing"
+)
+
+//TODO(abhide): Add more testcases once BuildV6Restore() are implemented
+func Test_BuildV6Restore(t *testing.T) {
```

Comment:
Don't use underscores in method names:
```suggestion
func TestBuildV6Restore(t *testing.T) {
```
Same for all other functions below.

Reply:
done